proof/refine/ARM/Ipc_R.thy

```diff
@@ -1483,8 +1483,75 @@ lemma make_arch_fault_msg_corres:
      apply (wp+, auto)
   done
 
+lemma schedContextUpdateConsumed_corres:
```

Comment:
After looking at Mitch's PR ("activate_thread...") it would be much better to factor out the `sc_consumed_update/ scConsumedUpdate` corres rule to reduce the duplication in this proof significantly. If we decide to merge my PR first, I will do that